### PR TITLE
`build`, `chroot`: add `-I <pkg>`, and also `aur build --cargs` just in case

### LINKED ISSUE #1124: `chroot`, `build`: add a flag to install a package into the build chroot
It would be nice if `aur build` (and, by extension, `aur chroot`) had a way to pass an `-i <pkg>` flag to makechrootpkg (yes, `aur chroot` has `--cargs`, but `aur build` has nothing).

This could be useful to install extra packages (e. g. ccache or sccache) into the temporary chroot when `-c -T` is used.

### PR DESCRIPTION
**Pull request by intelfx**
Add support for the `-I|--install <pkg>` flag to both `aur build` and `aur chroot` and pass this flag through to `makechrootpkg` (in makechrootpkg, it is a newly added flag `-i`).

Furthermore, add `aur build --cargs` (and pass this flag through to `aur chroot`) just in case there are future `makechrootpkg` flags we will not have exposed.

Depends on https://gitlab.archlinux.org/archlinux/devtools/-/merge_requests/214.

Fixes #1124.

### CODE REVIEW

**Comment by intelfx** [comment]
Welp -- nevermind, `makechrootpkg -I` does a totally different thing, namely installs a package _file_...

I guess devtools needs a new flag for this usecase

**Comment by intelfx** [comment]
Alright, this now depends on https://gitlab.archlinux.org/archlinux/devtools/-/merge_requests/214.

File: lib/aur-build [hunk 1]
```diff
@@ -147,6 +148,8 @@ while true; do
             chroot_args+=(--temp) ;;
         -I|--install)
 			shift; chroot_args+=(--install "$1") ;;
+		--cargs|--makechrootpkg-args)
```
Comment (AladW) [inline lib/aur-build:151]: The indentation seems off here

**Comment by AladW** [comment]
https://github.com/aurutils/aurutils/pull/1125/commits/c8658e5fd581789869101f1c5306909315f67d57 and https://github.com/aurutils/aurutils/pull/1125/commits/47d9b36ed2baf129e7db68c91b8653c8cefdcfd0 look fine for merging though (formatting aside). The `changelog` should also be updated.

**Comment by intelfx** [comment]
@AladW yeah, assuming I won't be able to convince devtools maintainers -- if you are sympathetic to the use-case described here, maybe this could be the final straw to reimplement makechrootpkg functionality within aurutils? What do you think?

**Comment by intelfx** [comment]
I can go through #909 and copy-write mkarchroot and makechrootpkg as e. g. `aur-chroot--create` and `aur-chroot--build`, applying points from #909 where relevant, and then we can turn the rest of it into separate issues...

**Comment by AladW** [comment]
I don't think it makes sense to copy makechrootpkg and mkarchroot entirely, since I don't want to copy all of the legacy design and implementation issues. Some ideas can be recycled though. See the `chroot-standalone` branch.

**Comment by AladW** [comment]
Since a devtools replacement is in the works (#1155), I think this PR can be closed in favor of that one.

**Pull request closed without merging**